src/Suave/Suave.fsproj

```diff
@@ -140,15 +179,7 @@
     <Reference Include="System.Web.Extensions" />
     <Reference Include="System.Xml" />
   </ItemGroup>
-  <Choose>
-    <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And $(TargetFrameworkVersion) == 'v4.5'">
-      <ItemGroup>
-        <Reference Include="FSharp.Core">
-          <HintPath>..\..\packages\FSharp.Core\lib\net40\FSharp.Core.dll</HintPath>
-          <Private>True</Private>
-          <Paket>True</Paket>
-        </Reference>
-      </ItemGroup>
-    </When>
-  </Choose>
+  <Target Name="BeforeBuild" BeforeTargets="Build" Condition="'$(TargetPlatform)' == 'coreclr'">
+    <Exec Command="..\..\tools\nuget\NuGet.exe restore -Source https://www.myget.org/F/dotnet-core;https://www.myget.org/F/fsharp-daily;https://nuget.org/api/v2 project.json -PackagesDirectory ..\..\packages -Verbosity detailed" />

```

Comment:
with the current design of dnx/coreclr it seems to be hopeless to enable Paket. I wrote about some of the issues [here](https://github.com/fsprojects/Paket/issues/1230#issuecomment-157324351). Basically it's merge conflict hell or hell of deploying quasi-random versions. But there is light on the end of the tunnel. If MS decides to build their "snapshot" files in coreclr/DNX then everything could work beautifuly together. See https://github.com/fsprojects/Paket/issues/1230#issuecomment-157343167 - let's just hope they do that before they go RTM.